src/mps/projmpo.jl

```diff
@@ -1,19 +1,44 @@
 
+"""
+A ProjMPO object represents the projection of an
+MPO into a basis defined by an MPS, leaving a
+certain number of site indices of the MPO unprojected.
+Which sites are unprojected can be shifted by calling
+the `position!` method.
+
+Drawing of the network represented by a ProjMPO `P`, 
+showing the case of `nsite(P)==2` and `position!(P,4)`:
```

Comment:
Isn't there only a function `position!(pm::ProjMPO, psi::MPS, pos::Int)`, not `position!(pm::ProjMPO, pos::Int)`?

Reply:
Yes, fixed this in new commits